src/spicy/manager.h

```diff
@@ -354,7 +357,7 @@ class Manager : public zeek::plugin::Plugin {
         bool operator==(const ProtocolAnalyzerInfo& other) const {
             return name_analyzer == other.name_analyzer && name_parser_orig == other.name_parser_orig &&
                    name_parser_resp == other.name_parser_resp && name_replaces == other.name_replaces &&
-                   protocol == other.protocol && linker_scope == other.linker_scope;
+                   protocol == other.protocol && ports == other.ports && linker_scope == other.linker_scope;
```

Comment:
Not sure that matters, but `port 21/tcp, port 22/tcp` is different from `port 22/tcp, port 21/tcp`. Wonder if the parsing code should sort/normalize?

Reply:
Good point, will do.